src/components/YourCompany/FolioInformation.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <FolioNumber
+    :initialValue="getFolioNumber"
+    :originalValue="originalFolioNumber"
+    :editLabel="editLabel"
+    :editedLabel="editedLabel"
+    :disableActions="isCorrectionFiling"
+    :invalidSection="invalidSection"
+    @newFolioNumber="onNewFolioNumber($event)"
+    @haveChanges="onHaveChanges($event)"
+    @isEditing="onIsEditing($event)"
+  />
+</template>
+
+<script lang="ts">
+import { Component, Emit, Mixins, Prop, Vue } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF, IncorporationFilingIF } from '@/interfaces'
+import { AuthApiMixin, CommonMixin } from '@/mixins'
+// TODO: move FolioNumber to bcrs-shared-components (#5024 part 2)
+import FolioNumber from './FolioNumber.vue'
+
+@Component({
+  components: { FolioNumber }
+})
+export default class FolioInformation extends Mixins(AuthApiMixin, CommonMixin) {
```

Comment:
This component acts as a shim/layer between the parent (main template stuff) and the low-level FN component (which will be shared eventually).
This component intentionally looks a lot like the Business Contact Info component.

Reply:
This component abstracts its details from its parent, while also interfacing with the store and services, so the FN component doesn't have to.